src/cdk/tree/control/flat-tree-control.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {BaseTreeControl} from './base-tree-control';
+
+/** Flat tree control. Able to expand/collapse a subtree recursively for FlatNode type. */
```

Comment:
What is `FlatNode type`?

Reply:
`FlatNode` type is deleted. Updated to the doc.